lib/streamlit/MediaFileManager.py

```diff
@@ -94,18 +99,22 @@ class MediaFileManager(object):
     def __init__(self):
         self._files = {}
         self._session_id_to_file_ids = collections.defaultdict(
```

Comment:
This name isn't correct anymore. This is now a mapping of session_id to dict of coordinates to file_id.

So how about something like:
```py
# Mapping of session ID to dict of coordinates -> file ID
self._session_coordinate_id_map = ...
```

Reply:
sure